### DISCUSSION TITLE
Bubble chat spacing?

### DISCUSSION THREAD

**Discussion by itaishalom** [flyerhq/flutter_chat_ui#138] [Q&A] [answered]
Hi, can you explain to me why is the spacing is different between the bubbles?
![Screen Shot 2021-10-25 at 18 29 24](https://user-images.githubusercontent.com/9066121/138725539-33f60e7f-af0b-4cc3-ab27-a8648b3a6f26.png)

**Comment by dariakhimych** [comment, marked as answer]
Hey! @itaishalom These are the grouped messages. The spacing between messages depends on the time gap between them. This feature is fully configurable. Just look up for these variables:

`dateHeaderThreshold` = 900000 ms (or 15 min)
`groupMessagesThreshold` = 60000 ms (or 1 min)

Hope it was helpful! ✌️

**Reply by itaishalom** [reply]
Awesome, thank you!

**Reply by itaishalom** [reply]
@dariakhimych can it be changed? I want different hights

**Reply by demchenkoalex** [reply]
Heights are not customisable AFAIR, because of the structure of the project, some of the height is in the message itself, the other is in the divider component etc. and everything is calculated to be symmetrical

**Reply by itaishalom** [reply]
Hmm, Ok, thank you!
